core/frontend/src/components/video-manager/VideoStream.vue

```diff
@@ -20,60 +15,43 @@
           Status: {{ stream.running ? 'running' : 'Not running' }}
         </p>
       </div>
-      <v-simple-table
-        dense
-        class="text-center"
-        style="margin: auto;"
-      >
-        <template #default>
-          <tbody>
-            <tr>
-              <td>{{ stream.video_and_stream.stream_information.configuration.encode }}</td>
-            </tr>
-            <tr
-              v-for="(endpoint, index) in stream.video_and_stream.stream_information.endpoints"
-              :key="index"
-            >
-              <td>
-                {{ endpoint }}
-              </td>
-            </tr>
-            <tr>
-              <td>{{ source_path }}</td>
-            </tr>
-          </tbody>
-          <v-btn
-            v-if="isSDPFileAvailable"
-            class="ma-6 elevation-1"
-            small
-            @click="downloadSDPFile"
-          >
+      <div class="d-flex ma-6 flex-column" :class="{ 'align-right': $vuetify.breakpoint.xs }">
+        <v-simple-table dense class="text-center" style="margin: auto">
+          <template #default>
+            <tbody>
+              <tr>
+                <td>{{ stream.video_and_stream.stream_information.configuration.encode }}</td>
+              </tr>
+              <tr v-for="(endpoint, index) in stream.video_and_stream.stream_information.endpoints" :key="index">
+                <td>
+                  {{ endpoint }}
+                </td>
+              </tr>
+              <tr>
+                <td>{{ source_path }}</td>
+              </tr>
+            </tbody>
+          </template>
+        </v-simple-table>
+        <div v-if="isSDPFileAvailable" class="ma-2 justify-space-between">
+          SDP file:
+          <v-btn v-tooltip="'Download to file'" class="ma-1 elevation-1" small @click="downloadSDPFile">
             <v-icon>mdi-download</v-icon>
-            SDP
           </v-btn>
-        </template>
-      </v-simple-table>
+          <v-btn v-tooltip="'Copy to clipboard'" class="ma-1 elevation-1" small @click="copySDPFileURL">
```

Comment:
`Copy file URL to clipboard`

Reply:
True, it's not clear if it's the URL or the file content that would be copied. Maybe also change the icon to this one?

![image](https://user-images.githubusercontent.com/5920286/219777488-3f4305d4-fd0c-4541-bcee-469debbf9ae6.png)